redbeams/src/main/java/com/sequenceiq/redbeams/configuration/EndpointConfig.java

```diff
@@ -78,4 +76,8 @@ private void registerEndpoints() {
         register(io.swagger.jaxrs.listing.SwaggerSerializers.class);
         register(io.swagger.jaxrs.listing.AcceptHeaderApiListingResource.class);
     }
+
+    private void registerFilters() {
+        register(CDPRestAuditFilter.class);
```

Comment:
No, it is not needed because strict audit filter was the main goal of this change. That sends common data about the REST calls not domain specific what the CDPStructuredEventFilter is used for.

Reply:
Okay, thanks for the clarification!